gap/DocumentationTree.gi

```diff
@@ -632,9 +658,10 @@ end );
 ##
 InstallMethod( WriteDocumentation, [ IsTreeForDocumentationDummyNodeRep, IsStream ],
   function( node, filestream )
-    if IsBound( node!.content ) then
-        WriteDocumentation( node!.content, filestream );
+    if node!.level > ValueOption( "level_value" ) then
+        return;
```

Comment:
Technically, this can be done, it would just require to pass it to all functions.

Historically I would say somebody (i.e., me) was too lazy to do it properly, and pushing those values made it much easier.

Reply:
I created PR #219 to address this.